### PR TITLE
Restyle Assignable CTA

### PR DESCRIPTION
**Pull request by RoyEJohnson**
For: https://github.com/openstax/openstax.org/issues/2246

### CODE REVIEW

File: src/app/pages/details/common/resource-box/resource-box.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function resourceBoxPermissions({
 // is available to the user (instructor version)
 export function instructorResourceBoxPermissions(resourceData, userStatus, search) {
     const resourceStatus = () => {
-        if (resourceData.resourceUnlocked || userStatus.isInstructor) {
+        if (resourceData?.resource?.resourceUnlocked || userStatus.isInstructor) {
```
Comment (jivey) [inline src/app/pages/details/common/resource-box/resource-box.js:48]: I'm super unfamiliar with os-web, where does this nested resource come from? Is this related to the CTA change?

**Review by jivey — COMMENTED** [review]
CTA change looks fine to me, is there a related card I can peek at to compare?

File: src/app/pages/details/common/resource-box/resource-box.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/pages/details/common/resource-box/resource-box.js:48]: No, it's a tacked-on fix for something Ed pointed out the other day. I'll put it in a separate PR.
image

**Comment by RoyEJohnson** [comment]
@jivey You can compare [production](https://openstax.org/assignable) to [Dev](https://dev.openstax.org/assignable) (I've just re-installed this branch in Dev).

File: src/app/pages/details/common/resource-box/resource-box.js (hunk 1 above)
Comment (jivey) [reply src/app/pages/details/common/resource-box/resource-box.js]: Ah thanks for the clarification. I'll probably have lots of questions as I get to know the codebase :)

**Review by jivey — APPROVED** [review]
